docs/content/tools/xdebug.md

```diff
@@ -4,210 +4,177 @@ aliases:
+    
+    ![PHPStorm Xdebug Drush](/images/xdebug-phpstorm-drush.png)
 
-### File Mapping Drush 9.x
+You can run your scripts in console and debug them in the same way as browser requests. For example, you can run 
+`fin drush fl` and debug this Drush command from the Features module.
 
-- Add `"/usr/local/bin/drush": "${workspaceRoot}/vendor/bin/drush"` to `pathMappings` in your `launch.json` file.
 
-Here is an an example of what `launch.json` should look like:
-```json
-{
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Listen for XDebug",
-      "type": "php",
-      "request": "launch",
-      "port": 9000,
-      "pathMappings": {
-        "/var/www/": "${workspaceRoot}",
-        "/var/www/vendor/bin/drush": "${workspaceRoot}/bin/drush"
-      }
-    },
+## Debugging with Visual Studio Code {#vscode}
+
+1. Follow the [setup instructions](#setup) to enable the Xdebug integration
+2. Install [PHP_Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) plugin by Felix Becker
+3. Configure PHP Debug Settings
+    
+    - Click Debug Icon in left sidebar
+    - Click Configuration Gear in top bar, select `PHP` from the dropdown list
+    - Add path mappings should be configured properly
+    
+    ![Visual Studio Xdebug](/images/xdebug-vscode.png)
+    
+    Here is an an example of what `launch.json` should look like:
+    
+    ```json
     {
-      "name": "Launch currently open script",
-      "type": "php",
-      "request": "launch",
-      "program": "${file}",
-      "cwd": "${fileDirname}",
-      "port": 9000
+      "version": "0.2.0",
+      "configurations": [
+        {
+          "name": "Listen for XDebug",
+          "type": "php",
+          "request": "launch",
+          "port": 9000,
+          "pathMappings": {
+            "/var/www/": "${workspaceRoot}"
+          }
+        },
+        {
+          "name": "Launch currently open script",
+          "type": "php",
+          "request": "launch",
+          "program": "${file}",
+          "cwd": "${fileDirname}",
+          "port": 9000,
+          "pathMappings": {
+            "/var/www/": "${workspaceRoot}"
+          }
+        }
+      ]
     }
-  ]
-}
-```
-- Ensure that your docksal.env or docksal-local.env has been updated to include `XDEBUG_ENABLED=1`.  This is off by default due to performance.
-- If changes were made, be sure to run `fin up` to update your environment.
-
-Here is an an example of what `docksal-local.yml` should include:
-```yaml
-version: "2.1"
-services:
-  cli:
-    environment:
-      - PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}
-```
-- Use `fin exec /var/www/vendor/bin/drush <site alias> <drush command>` to run the command you want to debug.
+    ```
+    
+4. Set a breakpoint
+5. Select the **Listen for XDebug** configuration from the dropdown and click **Start Debugging**
+
+You can debug both, web requests and cli scripts, using this configuration. 
```

Comment:
You can debug both web requests and cli scripts using this configuration.

Reply:
Thanks, updated